Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -121,6 +121,12 @@ extension TuistGraph.Target {
                 return []
             }
         }
+        
+        // remove excluding
+        manifest.resources?.excluding.forEach { path in
+            allResources.remove(path: AbsolutePath(path.pathString))
+        }
```

Comment:
As for `Sources`, I guess excluding could be a `glob`, so we should handle that as well